app/serializers/v1/task_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module V1
+  class TaskSerializer < ActiveModel::Serializer
+
+    attributes :id, :name, :description, :state
```

Comment:
Tasks come within the context of a `Project` (i.e. you have to use a `project_id` to do anything with tasks), so it's reasonable to not include it in the serialization, however it might be worth doing so just for thoroughness.